### PR TITLE
Update external-email-forwarding.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected the definition of automatic to  This is System-controlled: Allows outbound spam filtering to control automatic external email forwarding. This is the default value. as per https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/configure-the-outbound-spam-policy?view=o365-worldwide

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3056

File: microsoft-365/security/office-365-security/external-email-forwarding.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ description: "."
 
 External forwarding is controlled by the *outbound anti-spam policy* and scoped to users based on the configured setting. Currently 3 settings are supported:
 
-- **Automatic** – In this mode the system is responsible for deciding if a forwarded message is allowed or not.  This is the default mode and in this mode the system will block automatic external forwarding.
+- **Automatic** – This is System-controlled: Allows outbound spam filtering to control automatic external email forwarding. This is the default value.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/external-email-forwarding.md]:
```suggestion
- **Automatic** – This is system-controlled: It allows outbound spam filtering to control automatic external email forwarding. This is the default setting.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**